src/geometry/CTiglIntersectionCalculation.cpp

```diff
@@ -476,5 +476,10 @@ const std::string& CTiglIntersectionCalculation::GetID()
     return id;
 }
 
+const TopoDS_Compound& CTiglIntersectionCalculation::GetIntersectionResult()
```

Comment:
Why was this added? Is this needed?

Reply:
I think this is a leftover of a former attempt to restructure the CTiglIntersectionCalcualtion class and implement the tiglGetCrossSectionArea function in a different way. It's not needed in the current implementation, so I remove it.